packages/@react-aria/overlays/src/usePreventScroll.ts

```diff
@@ -13,19 +13,32 @@
 import {useEffect} from 'react';
 
 /**
- * Prevents scrolling on the document body on mount, and 
+ * Prevents scrolling on the document body on mount, and
  * restores it on unmount. Also ensures that content does not
  * shift due to the scrollbars disappearing.
  */
-export function usePreventScroll(): void {
+
+interface PreventScrollOptions {
+  isOpen?: boolean
+}
+
+export function usePreventScrollr(options?: PreventScrollOptions) {
+  const {isOpen = true} = options;
```

Comment:
this will change the default behavior, it used to be default false

Reply:
Ithanks @snowystinger , I'm currently working on it..